assets/sass/components/data.scss

```diff
@@ -148,7 +148,7 @@
             width: 160px;
             position: absolute;
             z-index: 10;
-            right: -$spacingUnit;
+            right: 0;
```

Comment:
This resolves this issue:

![image](https://user-images.githubusercontent.com/394376/32790441-60b98200-c956-11e7-84bf-c146fdba9fc8.png)

(eg. on the right is the current data map box, being moved offscreen)